replay_analyser.py

```diff
@@ -76,6 +76,23 @@ def team_rating(teams, replay_team):
              'red team': mean(rating for rating, tier in teams[1 - replay_team])})
 
 
+def custom_get_tank(tankName, tank_info):
+    tier = tank_info.get(tankName, {}).get('tier') # first try exact match
+    # if that fails try endswith matching
+    if tier is not None:
+        return tier
+
+    for tank in tank_info:
+        if (tank.endswith(tankName)):
+            return tank_info.get(tank).get('tier')
```

Comment:
How well does this work in matching tanks?  Do you have any data on how well it matches?
The key for each tank is the tank's tag - line 80 here
`tier = tank_info.get(tankName, {}).get('tier')`
will find it.

So we will either not get here, or we will search all known tanks and not find a match.